cst438_register/src/main/java/com/cst438/controller/StudentController.java

```diff
@@ -17,6 +18,7 @@
 import com.cst438.service.GradebookService;
 
 @RestController
+@CrossOrigin(origins = {"http://localhost:3000", "https://registerf-cst438.herokuapp.com/"})

```

Comment:
Ha, pretty cool you were trying to use your deployed database dev instance.

Reply:
I actually wasn't I believe this is from the original source repo, I assumed it was for testing/grading.